docs/community_snippets.md

```diff
@@ -292,6 +293,24 @@ A simple PSCustomObject by @brettmillerb. It has 4 properties that you can tab t
 }
 ```
 
+### Region Block
+
+Use the `#region` for organizing your code (including good code folding).
+
+#### Snippet
+
+```json
+"Region Block": {
+    "prefix": "#region",
+    "body": [
+        "#region ${1}",
+        "${2}",
```

Comment:
Do people typically put the region tag after `#endregion`?

I've always seen:

```
#region foo

#endregion
```

vs

```
#region foo

#endregion foo
```

Reply:
I'm not sure if `#endregion My cool region` is supported (it might be, but I've never seen it).

In C# the most common form I've seen is:
```csharp
#region Private variables

private readonly IEnumerable<string> _strs = new [] { "hello", "goodbye" };

private string _greeting = "Aloha";

#endregion // Private variables
```

I would imagine the PowerShell version of that is:

```powershell
#region My region
...
#endregion # My region
```

Reply:
I guess in PowerShell the region is already commented out, but that's an implementation detail from the line-comment delimiter and the pragma start-char being the same in my mind.

Reply:
My first pref (by a smidge) would be:
```
#endregion
```
Second would be:
```
#endregion <name>
```
But both would be fine by me.

Reply:
I feel like if both @tylerl0706 and @rkeithhill agree on a first preference then it should be that 😄

Reply:
✓